src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -388,11 +388,22 @@ boolean refersToImpl(T obj) {
     private native boolean refersTo0(Object o);
 
     /**
-     * Clears this reference object.  Invoking this method will not cause this
-     * object to be enqueued.
+     * Clears this reference object. Invoking this method does not enqueue this
+     * object, and the garbage collector will no longer enqueue this object once
+     * the referent reaches the designated reachability level.
```

Comment:
I'm wondering if "designated reachability level" is the right words to use here. The Notification section in the package description speaks of when the reachability of the referent has changed to the value corresponding to the type of the reference and I wonder if it might be better to use wording consistent with that. Minimally it could link to the package description where the notion of reachability level is introduced.

Reply:
Perhaps it's sufficient to say simply that, _"the garbage collector will no longer enqueue this object."_

Reply:
Is this sentence needed?   Excluding the race scenario as described in the javadoc, it would be equivalent as if the Reference with null referent which will never get enqueued?

Reply:
I personally think it's worth clearly stating that calling `clear()` opts you out of getting the GC notification that you signed up for when you registering this Reference with a RefQ.